data_collection/gazette/spiders/ce_juazeiro_do_norte.py

```diff
@@ -0,0 +1,62 @@
+import datetime as dt
+
+import scrapy
+import re
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class CeJuazeiroDoNorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "2307304"
+    name = "ce_juazeiro_do_norte"
+    allowed_domains = ["juazeiro.ce.gov.br"]
+    start_date = None
+
+    def __init__(
+        self, start_date: dt.date = None, end_date: dt.date = None, *args, **kwargs
+    ):
+        self.base_url = "https://juazeiro.ce.gov.br"
+        self.start_date = dt.date(year=2009, month=1, day=1)
+        self.end_date = dt.date.today()
+
+        super(CeJuazeiroDoNorteSpider, self).__init__(start_date, end_date)
+
+        self.logger.debug(
+            "Start date is {date}".format(date=self.start_date.isoformat())
+        )
+        self.logger.debug("End date is {date}".format(date=self.end_date.isoformat()))
+
+    def start_requests(self):
+        """
+        This base URL accepts two dates (beginning-date and end-date) or only one date (which will return
+        the gazettes for the given date). I will use the second one.
+        """
+        target_date = self.start_date
+        while target_date <= self.end_date:
+            formatted_date = target_date.strftime("%d-%m-%Y")
+            search_url = f"{self.base_url}/Imprensa/Diario-Oficial/{formatted_date}/"
+
+            yield scrapy.FormRequest(
```

Comment:
**question**: Why you chose a `FormRequest` instead of a `Request`? `yield scrapy.Request(url=search_url, meta={"date": target_date})` is not enough?

Reply:
Good point. No reason, I used other spider as template and forgot to change.
Corrected it now.